client/src/index.jsx

```diff
@@ -256,7 +258,16 @@ class App extends React.Component {
             <Route 
               path="/view?=(:id)" 
               render={HomeWithProps}
-              onEnter={ this.requireAuth }/>
+              onEnter={ this.requireAuth }
+              />
+            <Route 
+              path="/:username" 
+              render={routeProps => <Profile {...routeProps} 
+                refreshUserData={this.refreshUserData.bind(this)}
+                isLoggedIn={this.state.isLoggedIn} 
+                logUserOut={this.logUserOut.bind(this)}
+                userInfo={this.state.userInfo} />} 
+              />
```

Comment:
Same as above. I think it's a stupid auto-indenting thing. VS Code does the same thing :(